scripts/effectResolver/effectResolverCustom.js

```diff
@@ -0,0 +1,97 @@
+import { getSearchString } from "../utils.js";
```

Comment:
Moved from `effectResolver.js` and expanded/rewritten to favor "per-actor" effects before "any actor" effects